soh/soh/Enhancements/crowd-control/CrowdControl.cpp

```diff
@@ -215,7 +225,7 @@ void CrowdControl::ProcessActiveEffects() {
                             EmitMessage(tcpsock, effect->id, effect->timeRemaining, EffectResult::Success);
                         }
                     }
-
+                    GameInteractor::ApplyEffect(effect->giEffect);
```

Comment:
ah, so this is where it's happening now

i still feel like it'd be nice to have the apply methods do the checking, but i'm very open to being convinced otherwise (there very well could be something i'm overlooking)